regression/cbmc-incr-oneloop/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 add_test_pl_tests(
-    "perl -e 'alarm shift @ARGV; exec @ARGV' 8 $<TARGET_FILE:cbmc> --slice-formula"
+    "perl ../timeout.pl 8 $<TARGET_FILE:cbmc> --slice-formula"
```

Comment:
"Wrongful passing" sounds a bit like something you'd say at an eulogy for someone who was run over by a drunk driver. :smile: 

Worth pointing out here is that this is again CMake issue; Semicolons are list-item separators in CMake. Passing strings with semicolons in them around in CMake is incredibly error prone, it'd be possible to work around this here with some extra escapes but this approach is better!

SHA: aaa8425c8d210fdf9fb645721c34b5861c878dfe